docs/learn/smart-contract-developers/create-nft-collection-with-lsp7-tokenId.md

```diff
@@ -0,0 +1,956 @@
+---
+sidebar_label: '🃏 Create NFT Collection with a contract address as tokenId'
+description: 'Learn how to create a NFT collection with a contract address as tokenId'
+sidebar_position: 8
+---
+
+# Create NFT Collection with a contract address as tokenId
+
+:::note
+
+This guide builds on top of a Hardhat project using TypeScript as described in the [Getting Started](./getting-started.md) section.
+
+:::
+
+## Introduction
+
+If you have been looking at our [LSP8 token standard](../../standards/tokens/LSP8-Identifiable-Digital-Asset.md), you probably have noticed that addresses could be used as `tokenIds`. In this guide, we will replicate a real-world example where this could be useful.
```

Comment:
Suggest some extra precisions

```suggestion
If you have been looking at the [LSP8 token standard](../../standards/tokens/LSP8-Identifiable-Digital-Asset.md), you probably noticed that addresses could be used as `tokenIds`. In this guide, we will replicate a real-world example where this could be useful to represent an NFT / `tokenId` as a smart contract with its own metadata.
```